profiler/README.rst

```diff
@@ -0,0 +1,7 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Odoo Profiler
```

Comment:
Use the Readme template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst